packages/framework/fluid-static/api-report/fluid-static.api.md

```diff
@@ -116,6 +116,18 @@ export interface IServiceAudienceEvents<M extends IMember> extends IEvent {
     (event: "memberRemoved", listener: MemberChangedListener<M>): void;
 }
 
+// @public
+export interface IServiceClient<TServices> {
+    createContainer<const TContainerSchema extends ContainerSchema>(containerSchema: TContainerSchema): Promise<{
+        container: IFluidContainer<TContainerSchema>;
+        services: TServices;
+    }>;
+    getContainer<const TContainerSchema extends ContainerSchema>(id: string, containerSchema: TContainerSchema): Promise<{
+        container: IFluidContainer<TContainerSchema>;
+        services: TServices;
+    }>;
+}
```

Comment:
@ChumpChief - Maybe 'IFluidClient'?

Reply:
I'm ok with IServiceClient - it's a service-specific client, so this is more descriptive I think.